frontend/src/component/playground/Playground/PlaygroundForm/PlaygroundConnectionFieldset/PlaygroundConnectionFieldset.tsx

```diff
@@ -38,6 +42,11 @@ interface IOption {
 
 const allOption: IOption = { label: 'ALL', id: '*' };
 
+const SmallClear = styled(Clear)({
+    height: '1.25rem',
```

Comment:
I haven't seen this way of styling icons in the codebase (with rems). We normally use theme.fontSizes.smallBody or sth similar.I'd check other usages of icons and follow existing idioms